apps/ejabberd/src/ejabberd.app.src

```diff
@@ -39,7 +39,8 @@
                   fast_tls,
                   usec,
                   uuid,
-                  xmerl
+                  xmerl,
+                  worker_pool
```

Comment:
I tried Poolboy for ejabberd_odbc refactor, and in big thests the `poolboy:transaction`-based flow was 100 times worse than random worker selection and queuing on process' message queue we had previously. I switched to worker_pool since it offers strategies that queue on message queue, too, and the mean queue time dropped ~20%